app/pages/about/about-home.spec.js

```diff
@@ -0,0 +1,23 @@
+/* eslint prefer-arrow-callback: 0, func-names: 0, 'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }] */
+
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import AboutHome from './about-home';
+
+describe('AboutHome', function () {
+  let wrapper;
+
+  before(function () {
+    wrapper = shallow(<AboutHome />);
+  });
+
+  it('renders without crashing', function () {
+    assert.equal(wrapper, wrapper);
```

Comment:
This is always going to be true. I'm pretty sure this test can just be an empty function, as the `before` block renders the component.